packages/TelegramClient-UI.package/TCUEmojiHelper.class/class/defaultDirectory.st

```diff
@@ -0,0 +1,6 @@
+default values
+defaultDirectory
+
+	^ (FileDirectory default directoryNamed: self defaultFolderName)
+			assureExistence;
```

Comment:
Is assureExistence really needed at this code position?

Reply:
it is not.